src/components/Tag/__tests__/Tag.spec.tsx

```diff
@@ -85,11 +86,11 @@ describe('<Tag> component', () => {
     it('should invoke onDelete when user interacts with delete button by pressing Enter', async () => {
         const user = userEvent.setup();
         view = render(
-            <Tag onClick={onTagClick} onDelete={onDeleteClick}>
+            <Tag onClick={onTagClick} onDelete={onDeleteClick} closeLabel={closeLabel}>
                 {text}
             </Tag>
         );
-        const button = screen.getByRole('button', { name: '' });
+        const button = screen.getByRole('button', { name: `${closeLabel}` });
```

Comment:
`const button = screen.getByRole('button', { name: closeLabel });` no need for `${}`